Vizzuality/climate-watch
# Updates Rails version

To test:

```
bundle install
rails db:test:prepare
rspec
```

And also run the app and check that all looks good in your browser.
